editor/src/components/navigator/left-pane/github-pane/github-file-changes-list.tsx

```diff
@@ -96,29 +106,63 @@ export const GithubFileChangesList: React.FC<{
 
   return (
     <>
-      <UIGridRow padded variant='<----------1fr---------><-auto->'>
-        <div>
-          {count} file{count !== 1 ? 's' : ''} changed
-        </div>
-        <RevertButton
-          disabled={githubWorking}
-          text='Revert all'
-          onMouseUp={handleClickRevertAllFiles}
+      {showHeader && (
+        <Header
+          count={count}
+          revertable={revertable}
+          githubWorking={githubWorking}
+          onClickRevertAll={handleClickRevertAllFiles}
         />
-      </UIGridRow>
-      {list.map((i) => (
-        <UIGridRow key={i.filename} padded variant='<----------1fr---------><-auto->'>
-          <UIGridRow padded variant='|--16px--|<--------auto-------->'>
-            <GithubFileStatusLetter status={i.status} />
-            <Ellipsis title={i.filename}>{i.filename}</Ellipsis>
+      )}
+      {list.map((i) => {
+        const conflicting = conflicts?.includes(i.filename) || false
+        return (
+          <UIGridRow
+            key={i.filename}
+            padded
+            variant='<----------1fr---------><-auto->'
+            title={conflicting ? 'Potential conflicts' : i.filename}
+            style={{
+              gap: 2,
+              color: conflicting ? '#f00' : 'inherit',
+              cursor: conflicting ? 'help' : 'default',
+            }}
+          >
+            <UIGridRow padded variant='|--16px--|<--------auto-------->'>
+              <GithubFileStatusLetter status={i.status} />
+              <FlexRow style={{ gap: 2 }}>
+                <Ellipsis>{i.filename}</Ellipsis>
+                {conflicting && <WarningIcon color='error' />}
```

Comment:
I believe this should be using the `when` utility function, we try to avoid exploiting truthiness especially with `&&` and `||`.